### PR TITLE
Fixed module path issue

### PR DESCRIPTION
**Pull request by GuanYung**
This PR fixed the following error
ModuleNotFoundError: No module named 'backend'

### CODE REVIEW

**Review by ravikiranchollangi — CHANGES_REQUESTED** [review]
Please look at this: https://docs.python.org/3/reference/import.html#package-relative-imports

**Review by ravikiranchollangi — COMMENTED** [review]
Two comments.

File: backend/restapi_server.py [hunk 1]
```diff
@@ -5,8 +5,18 @@
...
 import sys
-from submodule.rs_device_manager import RsDeviceManager
-from submodule.rs_device import ModuleType
+
+# specific the path solve the module load error
+myDir = os.getcwd()
+sys.path.append(myDir)
+from pathlib import Path
+path = Path(myDir)
+print(str(path.parent.absolute()))
```
[the remark this answers is not in the document]
Comment (GuanYung) [reply backend/restapi_server.py]: the print statement is removed now. I was using it for debugging.

File: backend/restapi_server.py [hunk 2]
```diff
@@ -5,8 +5,18 @@
 import argparse
 import os
 import sys
-from submodule.rs_device_manager import RsDeviceManager
-from submodule.rs_device import ModuleType
+
```
[the remark this answers is not in the document]
Comment (GuanYung) [reply backend/restapi_server.py:8]: all the files was using the `from backend.submodule......` import statements
I was finding solution to fix the issue. At least for now it is an workaround. I need to learn more details about the import package/module in Python. Until then, I will find a proper fix.

below is the print out of command `npm start` for your reference

image

**Comment by GuanYung** [comment]
@ravikiranchollangi 
As at now, this is what I have. If you believe there is a better solution, I will need more time to figure out. 
Let me know if this PR works or you want a perfect fix for this issue.

**Pull request closed without merging**